src/amqp10_client_session.erl

```diff
@@ -649,6 +657,44 @@ translate_terminus_durability(none) -> 0;
 translate_terminus_durability(configuration) -> 1;
 translate_terminus_durability(unsettled_state) -> 2.
 
+translate_filters(#{}) -> undefined;
```

Comment:
As we cannot run the azure tests as part of ci I would like some local unit tests for this function